config/install/block.block.views_block__alert_banner_alert_banner_block.yml

```diff
@@ -4,6 +4,7 @@ dependencies:
   config:
     - views.view.alert_banner
   module:
+    - localgov_alert_banner
     - views
```

Comment:
I would try and replace it with.
````yaml
module:
    - views
enforced:
    module:
      - localgov_alert_banner
````
This is because, when a module is uninstalled, the config can still be left behind. This guarantees the config gets removed.

Reply:
Ah, thanks @andybroomfield , understood and updated.